addons/website/views/snippets/s_freegrid.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_freegrid" name="Free grid">
+    <section class="s_freegrid o_cc o_cc1 pt64 pb64">
+        <div class="container-fluid">
+            <div class="row o_grid_mode" data-row-count="14">
+                <div class="o_grid_item o_grid_item_image g-col-lg-1 g-height-8 col-lg-1 d-lg-block d-none o_snippet_mobile_invisible" style="z-index: 1; grid-area: 2 / 1 / 10 / 2; --grid-item-padding-y: 0px; --grid-item-padding-x: 0px;">
+                    <img class="img img-fluid mx-auto" src="/web/image/website.library_image_03" alt=""/>
+                </div>
+                <div class="o_grid_item o_grid_item_image g-col-lg-3 g-height-5 col-lg-3 d-lg-block d-none o_snippet_mobile_invisible" style="z-index: 2; grid-area: 1 / 2 / 6 / 5; --grid-item-padding-y: 8px; --grid-item-padding-x: 16px;">
+                    <img class="img img-fluid mx-auto" src="/web/image/website.library_image_13" alt=""/>
+                </div>
+                <div class="o_grid_item o_grid_item_image g-col-lg-3 g-height-5 col-lg-3 d-lg-block d-none o_snippet_mobile_invisible" style="z-index: 3; grid-area: 6 / 2 / 11 / 5; --grid-item-padding-y: 8px; --grid-item-padding-x: 16px;">
+                    <img class="img img-fluid mx-auto" src="/web/image/website.library_image_10" alt=""/>
+                </div>
+                <div class="o_grid_item o_grid_item_image g-col-lg-3 g-height-8 col-lg-3 d-lg-block d-none o_snippet_mobile_invisible" style="z-index: 4; grid-area: 2 / 5 / 10 / 8; --grid-item-padding-y: 0px; --grid-item-padding-x: 0px;">
+                    <img class="img img-fluid mx-auto" src="/web/image/website.library_image_05" alt=""/>
+                </div>
+                <div class="o_grid_item o_grid_item_image g-col-lg-3 g-height-10 col-lg-3" style="z-index: 5; grid-area: 1 / 8 / 11 / 11; --grid-item-padding-y: 16px; --grid-item-padding-x: 16px;">
```

Comment:
I don't think we need Y padding here

```suggestion
                <div class="o_grid_item o_grid_item_image g-col-lg-3 g-height-10 col-lg-3" style="z-index: 5; grid-area: 1 / 8 / 11 / 11; --grid-item-padding-y: 0px; --grid-item-padding-x: 16px;">
```

Reply:
Actually 8 would be better so it matches the alignement of top of image 2, bottom of image 3. wdyt?
```suggestion
                <div class="o_grid_item o_grid_item_image g-col-lg-3 g-height-10 col-lg-3" style="z-index: 5; grid-area: 1 / 8 / 11 / 11; --grid-item-padding-y: 8px; --grid-item-padding-x: 16px;">
```

Reply:
Indeed 👍